packages/core/src/store/index.js

```diff
@@ -55,6 +55,11 @@ export const aggregateStoreTableProps = {
       partitionKey: 'stat',
       sortKey: 'insertedAt',
       projection: 'all'
+    },
+    indexStorefront: {
+      partitionKey: 'storefront',
+      sortKey: 'insertedAt',
+      projection: 'all'
```

Comment:
added new index so that we can query all aggregates by a given storefront. We currently use this for the integration tests, given we create a new storefront did per integration test run, it will always be a new one.

This will likely not be needed in the day to day of operating this system. However, it might be a query important to also do ad hoc.